index.html

```diff
@@ -0,0 +1,341 @@
+                c0.96,0.54,2.02,0.81,3.19,0.81c2.1,0,3.64-0.97,4.63-2.9c0.99-1.93,1.48-4.95,1.48-9.06v-4.36
+                C179.36,29.84,179.17,27.43,178.8,25.69z M173.11,36.93c0,2.01-0.08,3.58-0.25,4.72c-0.16,1.14-0.44,1.95-0.83,2.43
+                c-0.39,0.48-0.91,0.72-1.57,0.72c-0.51,0-0.98-0.12-1.42-0.36c-0.43-0.24-0.79-0.6-1.06-1.08V27.71
+                c0.21-0.75,0.57-1.36,1.08-1.84c0.51-0.48,1.06-0.72,1.66-0.72c0.63,0,1.12,0.25,1.46,0.74c0.34,0.49,0.58,1.33,0.72,2.49
+                c0.13,1.17,0.2,2.83,0.2,4.99V36.93z"
+                    class="style-scope ytd-topbar-logo-renderer"
+                  ></path>
+                </g>
+              </g>
+            </g>
+          </svg>
+        </div>
+
+        <form class="search-form">
+          <input type="text" placeholder="Search" />
+          <button class="search-btn light-gray">
+            <i class="fas fa-search"></i>
+          </button>
+        </form>
+
+        <div class="nav-right-content flex-sb-center">
+          <img
+            src="assets/images/video-solid.svg"
+            alt=""
+            class="icon-svg right-margin"
+          />
+          <img
+            src="assets/images/th-solid.svg"
+            alt=""
+            class="icon-svg right-margin"
+          />
+          <img
+            src="assets/images/bell-solid.svg"
+            alt=""
+            class="icon-svg right-margin"
+          />
+
+          <img
+            class="user-image right-margin"
+            src="https://c0.wallpaperflare.com/preview/276/301/198/avatar-people-person-business.jpg"
+            alt="user_image"
+          />
+        </div>
+      </nav>
+    </header>
+
+    <main class="main">
+      <section class="main-left-content">
+        <h1 class="hidden-tags">Empty tags</h1>
+        <div class="video-player">
+          <iframe
+            class="responsive-iframe"
+            src="https://www.youtube.com/embed/d82URIOvgko"
+            allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
+            allowfullscreen
+          ></iframe>
+        </div>
+
+        <section class="video-details bottom-border">
+          <h3 class="hidden-tags">Empty tags</h3>
+          <p class="video-title">
+            Age Jodi Janitam Tobe Mon Phire Chahitam by Bappa Majumdar Partho
+            Barua Elita in Austin Unplugged
+          </p>
+          <div class="video-info flex-sb-center">
+            <p class="views light-gray">710,086 views • Feb 3, 2017</p>
+            <div class="icons flex-sb-center dark-gray">
+              <p class="right-margin">
+                <span><i class="fas fa-thumbs-up"></i></span> 42k
+              </p>
+              <p class="right-margin">
+                <span><i class="fas fa-thumbs-down"></i></span> 1k
+              </p>
+              <p class="right-margin">
+                <span><i class="fas fa-share"></i></span> Share
+              </p>
```

Comment:
Kindly the share should be a button, not an icon. You can add a button tag
![button](https://user-images.githubusercontent.com/32361963/83875363-9f8fc880-a73f-11ea-9c97-4f04b4cf05d6.png)